utilities/pom.xml

```diff
@@ -151,6 +151,13 @@
 			<groupId>org.jboss.resteasy</groupId>
 			<artifactId>resteasy-client</artifactId>
 		</dependency>
+
+		<dependency>
+			<groupId>com.dropbox.core</groupId>
+			<artifactId>dropbox-core-sdk</artifactId>
```

Comment:
Is there a smaller portion just with client?

Reply:
I don't think so